drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -408,7 +408,7 @@ func (k *k8sSchedOps) ValidateVolumeCleanup(d node.Driver) error {
 		Name:           "*portworx-volume",
 	}
 
-	for _, n := range node.GetWorkerNodes() {
+	for _, n := range node.GetStorageDriverNodes() {
```

Comment:
can you check if GetStorageDriverNodes return storage and storageless PX nodes (i.e nodes where PX is installed but are storageless)

Reply:
yes, it returns all workers and PX nodes:
```
if n.Type == TypeWorker && n.IsStorageDriverInstalled {
			nodeList = append(nodeList, n)
		}
```